### ISSUE TITLE
Clinica is too verbose + @clinica@ is displayed at the beginning of each cprint

### ISSUE THREAD

**Issue by alexandreroutier** [aramis-lab/clinica#7]
Hi !

I noticed that the terminal is now too verbose and that `@clinica@`  before each call to `cprint`.

In the past, the `dwi-dti` pipeline displayed:
<img width="1372" alt="OutputPipeline" src="https://user-images.githubusercontent.com/24808663/60167590-ac9ffa00-9803-11e9-85a0-47168258a4b8.png">

Now, the output looks like:
```
╰─ clinica run dwi-dti caps -np 1
@clinica@ FSL version must be greater than 5.0.5
@clinica@ FSL has been detected
@clinica@ ANTs has been detected
@clinica@ MRtrix has been detected
@clinica@ Found 1 image(s) in CAPS dataset
@clinica@ The pipeline will last approximately 20 minutes per image.
190626-11:12:03,452 nipype.workflow INFO:
	 Workflow DwiDti settings: ['check', 'execution', 'logging', 'monitoring']


190626-11:12:03,730 nipype.workflow INFO:
	 Running in parallel.


190626-11:12:03,737 nipype.workflow INFO:
	 [MultiProc] Running 0 tasks, and 4 jobs ready. Free memory (GB): 14.40/14.40, Free processors: 1/1.


190626-11:12:03,983 nipype.workflow INFO:
	 [Node] Setting-up "DwiDti.Read-CAPS_BVEC" in "/private/var/folders/m_/j76n37kn4vs6zsj8fq0qcgy0000dn9/T/tmp3nkrjugq/DwiDti/_session_id_ses-M00_subject_id_sub-CAPP01001TMM/Read-CAPS_BVEC".


190626-11:12:03,997 nipype.workflow INFO:
	 [Node] Running "Read-CAPS_BVEC" ("nipype.interfaces.io.DataGrabber")


190626-11:12:04,71 nipype.workflow INFO:
	 [Node] Finished "DwiDti.Read-CAPS_BVEC".
[...]
```

I think it is related to the redirection of the output when Clinica crashs.

If you know how to easily fix this issue, it would be nice. Otherwise, I will investigate this later.

Best,
Alex

**Comment by arnaudmarcoux** [comment]
When using `cprint` I think this is normal behavior as far as I remember

**Comment by alexandreroutier** [comment]
> When using `cprint` I think this is normal behavior as far as I remember

It is not the case (cf screenshot on my first message). Anyway, this will be solved when the PR #8 will be merger into Clinica ;-)

**Comment by alexandreroutier** [comment]
cf PR #8
